Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -73,5 +75,12 @@ void main (void)
     l = min(l, czm_luminance(rgb));
 #endif
 
-    gl_FragColor = vec4(rgb, min(smoothstep(0.0, 0.1, l), 1.0) * smoothstep(0.0, 1.0, czm_morphTime));
+    // Alter alpha based on how close the viewer is to the ground (1.0 = on ground, 0.0 = at edge of atmosphere)
+    float atmosphereAlpha = clamp((u_cameraAndRadiiAndDynamicAtmosphereColor.y - length(czm_viewerPositionWC)) / (u_cameraAndRadiiAndDynamicAtmosphereColor.y - u_cameraAndRadiiAndDynamicAtmosphereColor.z), 0.0, 1.0);
```

Comment:
`length(czm_viewerPositionWC)` can be replaced by `u_cameraAndRadiiAndDynamicAtmosphereColor.x` (camera height)